src/odemis/acq/milling/fibsemos.py

```diff
@@ -156,7 +188,8 @@ def __init__(self, future: futures.Future,
         :param path: The path to save the images (optional)
         """
         # create microscope connection
-        self.microscope = create_openfibsem_microscope()
+        # TODO: Eventually use create_fibsemos_tescan_microscope() once it can get host_ip/port from the model
+        self.microscope = create_fibsemos_thermo_microscope()
```

Comment:
You might want to create a new generic function in this file that creates the right fibsemOS microscope automatically.